GameOfLifeConsole/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace GameOfLifeConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            uint MaxRuns = 10;
+            int runs = 0;
+
+            int[,] seed = new int[,]
```

Comment:
So you don't have to maintain multiple projects maybe you can combine them by having some kind of selector of game kind that user selects in console.
For example when starting game user has option to select whether he wants to see blinker or randomly generated field.

Reply:
Done, using switch operator. the second project has been deleted. The main class with the logic game has been remained and UI for random and manual has been input in console.